0_Fib/fib.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func Fibonacci(n int) []int {
+
+	fib := []int{}
+	t1 := 0
+	t2 := 1
```

Comment:
You can do a one-liner: `t1, t2 := 0, 1`.

Reply:
done